charts/cluster-addons/values.yaml

```diff
@@ -340,6 +340,46 @@ nvidiaGPUOperator:
         serviceMonitor:
           enabled: true
 
+intelDevicePlugin:
+  enabled: false
+  operator:
+    chart:
+      repo: https://intel.github.io/helm-charts
+      name: intel-device-plugins-operator
+      version: 0.32.0
```

Comment:
We need to tweak the update system so we pick up changes in all these versions:
https://github.com/azimuth-cloud/capi-helm-charts/blob/dd78906a9939932ecff01ffe254b344a37b3a967/.github/workflows/update-addons.yml#L69